examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,616 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        "wait_before_acquire", 
</pre><p>Can we call it something more comprehensible?</p><p> </p><p>"Standing by" ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
<p>Standing by looks way more cryptical to me.</p>